src/components/ImportJobs/components/FileUpload/FileUpload.css

```diff
@@ -16,3 +16,14 @@
   margin-bottom: 0.5rem;
   font-weight: 600;
 }
+
+
+.activeUpload {
+  background: black;
+  color: #fff;
+  border-style: dashed;
+}
+
+.activeUploadTitle {
+  font-size: 30px;
```

Comment:
For font-size use `rem` unit. Firstly check whether such variable exists in variables file.